nvflare/lighter/project.yml

```diff
@@ -35,6 +35,9 @@ builders:
     args:
       # config_folder can be set to inform NVFlare where to get configuration
       config_folder: config
+      # config_validator is used to verify if uploaded app has proper structures
```

Comment:
Call it app_validator, not config_validator.

Reply:
Same as the above.  This was renamed.